.github/ISSUE_TEMPLATE/add-instance.yaml

```diff
@@ -6,32 +6,59 @@ labels:
+
+  - type: checkboxes
+    attributes:
+      label: Requirements
+      description: |
+        If you don't respect all the requirements your instance can't be added.
+        And if later after we add your instance, we found out that your instance fail to respect all requirements, we will remove it.
       options:
       - label: "It is my instance."
         required: true
-      - label: "I give the right to check.searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests)."
+      - label: "I give the right to `check.searx.space` to check my instance (every 3 hours for the response times, every 24 hours for the other tests)."
         required: true
-      - label: "I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)"
+      - label: "I acknowledge that managing a public instance is not an easy task and require spending time to keep the instance in good health. E.g. look after your instance by using a monitoring system."
         required: true
-      - label: "I guarantee to keep an uptime per month of my instance at **minimum 90%**. Your instance will be removed if you don't respect this rule."
+      - label: "I guarantee to keep an uptime per month of my instance at **minimum 90%**. Please ask for a removal of your instance if there is a planned long downtime."
+        required: true
+      - label: "I do not track the users of my instance with any analytics or tracking software."
+        required: true
+      - label: "I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)"
         required: true
 
+  - type: markdown
+    attributes:
+      value: |
+        ### Recommendations for your instance
+
+        - Try to not use Cloudflare or any other commercial solution that analyze the HTTP traffic between the user and the server.
+          It's bad for the privacy because the provider can see all the HTTP traffic between the users and your server.
+          Prefer using [the built-in limiter plugin](https://docs.searxng.org/src/searx.plugins.limiter.html#limiter-plugin) for SearxNG or [filtron](https://github.com/searxng/filtron) for SearX.
+        - Don't block JavaScript users with CAPTCHA that only work with JavaScript, page that verify the user with JavaScript and so on.
+        - Try to harden the TLS security of your server by getting an A+ TLS grade on https://cryptcheck.fr.
+          You can use https://ssl-config.mozilla.org to get A+ TLS grade.
+        - Try to harden the security of your users when they use your instance by getting an A+ CSP grade on https://observatory.mozilla.org.
+          Look into https://github.com/searxng/searx-docker/blob/master/Caddyfile#L33-L84 to get A+ CSP grade.
```

Comment:
We do not have a clear recommendation about CSP, I think it is OK to link to the caddy file of the docker image but we also have other resources where we have CSP

https://github.com/searxng/searxng/blob/7bf4e8d12d1d0ee53bf71f7c3a4010ef936f25d9/searx/settings.yml#L72-L77

IMO it is better we have a recommended CSP setup in settings.yml .. but this might be out of the scope of this PR

Reply:
I've no idea what to recommend for the CSP, I'll keep the link to the caddyfile like it was already previously but feel free to recommend another one if you want.

Reply:
> but feel free to recommend another one if you want.

I think we should address this topic in the SearXNG project .. but here it is OK to use the link to caddy ..